docs/site/src/components/ProtocolConfig/index.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useState, useEffect } from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import axios from "axios";
+
+export default function ProtocolConfig() {
+  const data = {
+    jsonrpc: "2.0",
+    id: 1,
+    method: "sui_getProtocolConfig",
+    params: [],
+  };
+  const urls = [
+    "https://fullnode.mainnet.sui.io:443",
+    "https://fullnode.testnet.sui.io:443",
+    "https://fullnode.devnet.sui.io:443",
+  ];
+  const [results, setResults] = useState({
+    mainnet: null,
+    testnet: null,
+    devnet: null,
+  });
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const parseResult = (data) => {
+    let items = Object.entries(data);
+
+    return items.map((item) => {
+      if (item[1] === null) {
+        return item;
+      }
+      if (typeof item[1] === "object") {
+        const [k, v] = Object.entries(item[1])[0];
+        return [item[0], k, v];
```

Comment:
Is object a JavaScript thing or is this part of the data returned by the jsonrpc call

Reply:
The data returned is a json object from the call. One of two possible entries is `{"name": { type: "value"}` or `{"name": null}`. I just double-check that the second item is in fact an object so the Object.entries doesn't freak out.

Reply:
Sorry, at this point it's either `["name", {type: "value"}]` or `["name", null]`